src/components/reps/preformatted-text-items-handler.jsx

```diff
@@ -1,6 +1,12 @@
 import React from "react";
+import styled from "react-emotion";
 import PropTypes from "prop-types";
 
+const TextItems = styled("pre")`
```

Comment:
this also standardizes the rep to have the same margin as the other outputs.

Reply:
is there anything that should be defined in THEMES and pulled from that source if something about this being standardized across components?

Reply:
same response as one above